client/src/Molecules/ContestCard/ContestCardStyled.js

```diff
@@ -10,17 +10,57 @@ const ContestCardStyled = styled.div`
   flex-direction: column;
   justify-content: space-around;
   padding: 1.25rem 1.25rem 1.375rem 1.25rem;
-  border: solid 3px
+  border: solid
     ${(props) =>
       props.colorMotive === c.GREEN
         ? props.theme.positive400
         : props.theme.grey200};
+  border-width: ${(props) => (props.colorMotive === c.GREEN ? '3px' : '1px')};
   margin: 0.5rem auto 0.5rem;
   background-color: ${(props) =>
     props.colorMotive === c.GREEN
       ? props.theme.positive100
       : props.theme.white};
   border-radius: 0.75rem;
+
+  .p, .doggo {
+    display: none;
+  }
+  
```

Comment:
I think You could use mobile_only class instead.

Reply:
I forget about it, I'll check it

Reply:
It's not good for me, I I just checked it out

Reply:
desktop_only works very well, className p could be more descriptive or in case of using global class not needed.